nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -238,6 +239,34 @@ public class PutDatabaseRecord extends AbstractProcessor {
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    static final AllowableValue BINARY_STRING_FORMAT_UTF8 = new AllowableValue(
+            "UTF8",
+            "UTF8",
+            "String values for binary columns contain the original value as text via UTF-8 character encoding"
+    );
+
+    static final AllowableValue BINARY_STRING_FORMAT_HEX_STRING = new AllowableValue(
+            "Hex String",
+            "Hex String",
+            "String values for binary columns contain the original value in hexadecimal format"
+    );
+
+    static final AllowableValue BINARY_STRING_FORMAT_BASE64 = new AllowableValue(
+            "Base64-encoded String",
+            "Base64-encoded String",
```

Comment:
Recommend using `Base64` to align with other options.
```suggestion
            "Base64",
            "Base64",
```